.github/workflows/ci.yml

```diff
@@ -128,7 +128,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - run: docker build --target builder .
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - run: docker buildx build --platform linux/amd64,linux/arm64 -f Dockerfile -t dotenvlinter/dotenv-linter .
```

Comment:
Any idea what changed since I've tried it? 
Because I think I did it the same way with docker build but it didn't work with our `Dockerfile`.

Reply:
I don't know 🤔
I wanted to ack you the same question 😅

Reply:
Can you continue your work in this PR https://github.com/dotenv-linter/dotenv-linter/pull/458?

Reply:
I think it's fine if you finish this one off here - at least if you want to, I can barely keep up with maintaining my own open-source projects 👍